src/blog/2023/01/flowforge-1-3-0-released.md

```diff
@@ -0,0 +1,86 @@
+
+<div><iframe width="560" height="315" src="https://www.youtube.com/embed/B7XK3TUklUU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+[Control access to your Node-RED dashboards using FlowForge credentials](https://github.com/flowforge/flowforge/issues/1325)
+In FlowForge 0.10 we added the ability to secure endpoints created within your FlowForge projects. This allows you to create dashboards or APIs and limit who can access them. In 1.3 we've added the ability for you to limit access to those same resources based on the visitor having a user account on your FlowForge team. You can see a demonstration of this new feature in [the video](https://youtu.be/JRk-Cf7eNIo) below.
+
+<div><iframe width="560" height="315" src="https://www.youtube.com/embed/JRk-Cf7eNIo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+[Filter your audit logs for easier reading](https://github.com/flowforge/flowforge/issues/1448)
+In FlowForge 1.3 we’ve added the ability to filter your admin logs by user or action type. We think this is a great new feature which will help admins have confidence that they will be able to review the audit logs quickly when needed. You can see a demonstration of this new feature in [the video](https://youtu.be/p0Vuy5x42Go) below.
+
+<div><iframe width="560" height="315" src="https://www.youtube.com/embed/p0Vuy5x42Go" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Improvements
+
+[Allow installation of FlowForge on devices which can't access npm](https://github.com/flowforge/flowforge-device-agent/issues/45)
+We've had feedback from customers that in some cases they want to use FlowForge devices on hardware which cannot access [Node Package Manager](https://www.npmjs.com/) (npm). In a standard configuration of Node-RED, access to npm is mandatory to run your flows. In FlowForge 1.3.0 we've added the ability for you to import all the data usually installed from npm without your devices having access to the npm service.
+
+## Bug Fixes
+
+We've fixed the following bugs in this release.
+
+- Project status UI sometimes getting stuck when restarting [#1232](https://github.com/flowforge/flowforge/issues/1232)
+
+- SSO users asked to click link in email to verify [#1543](https://github.com/flowforge/flowforge/issues/1543)
+
+- SSO users unable to edit settings [#1542](https://github.com/flowforge/flowforge/issues/1542)
+
+- SSO users not redirected to editor when signing in [#1481](https://github.com/flowforge/flowforge/issues/1481)
```

Comment:
Maybe a better style?

```suggestion
A couple of bugs were fixed in 1.3 too; one bug of those got the project status UI [stuck when restarting](https://github.com/flowforge/flowforge/issues/1232). A few around Single Sign in were fixed too; firstly one asking users to verify their email address, which isn't required with SSO. Another disallowed SSO users to their profile details, an unintended consequence from [disallowing these users to update their email address](https://github.com/flowforge/flowforge/issues/1542). Lastly there was a broken redirect [when signing in](https://github.com/flowforge/flowforge/issues/1481).
```

Reply:
Agreed, adding it in as part of my commit.